src/main/java/de/leonhard/storage/Config.java

```diff
@@ -56,6 +60,28 @@ public Config(final File file) {
     super(file);
   }
 
+  public void annotateClass(Object classInstance) {
+    this.annotateClass(classInstance, s -> "");
```

Comment:
The Idea here was to return s, if no prefixing should take place.
You can even use `Function.identitiy()` I think.